documentation/adding-new-rtypes.md

```diff
@@ -34,14 +34,12 @@ Here are some examples:
 
 ```go
 type RecordConfig struct {
-  ...
```

Comment:
We need a way to indicate that some fields in the struct have been removed for brevity

I agree with the removal of `...` in `D(X, ...` examples.  This is different.

How about:

```
    [...]
    field
    [...]
```

Reply:
> We need a way to indicate that some fields in the struct have been removed for brevity

That's a great suggestion, but I'd like to keep it out of the pull request.

> I agree with the removal of ... in D(X, ... examples.

I've applied a rebase commit that keeps the rest of the `...`.